### ISSUE TITLE
Duplicate `ClassNameMap` identifier in TypeScript definitions

### ISSUE THREAD

**Issue by Methuselah96** [iamhosseindhv/notistack#105]
## Expected Behavior

There should be no errors in the TypeScript definitions.

## Current Behavior

When compiling:
```
ERROR in [at-loader] ..\node_modules\notistack\index.d.ts:5:6 
    TS2300: Duplicate identifier 'ClassNameMap'.
ERROR in [at-loader] ..\node_modules\notistack\index.d.ts:32:13 
    TS2300: Duplicate identifier 'ClassNameMap'.
```

## Steps to Reproduce

Link:

1.
2.
3.
4.

## Context

Use `notistack` in a TypeScript app.

## Your Environment

| Tech         | Version |
|--------------|---------|
| Notistack | v0.6.0  |
| React        |         |
| Browser      |         |
| etc.         |         |

**Comment by Methuselah96** [comment]
Will be resolved by https://github.com/iamhosseindhv/notistack/pull/106.

**Comment by iamhosseindhv** [comment]
Fixed in #106 #104
